<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pam</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>pam_logo.jpg</normaloff>pam_logo.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>881</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Plot</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>861</width>
        <height>611</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,6,6" columnstretch="8,0">
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Display last</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="last_values1">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <number>5000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>values</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="refresh_x1">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Refresh X</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Plot1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="stop_plot1">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="start_plot1">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <widget class="QComboBox" name="QComboBox_1">
           <item>
            <property name="text">
             <string>Raw data</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>IRF</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Distance</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="5" column="0">
          <widget class="QComboBox" name="QComboBox_2">
           <item>
            <property name="text">
             <string>Raw data</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>IRF</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Distance</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Plot2</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="refresh_x2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Refresh X</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Display last</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="last_values2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <number>5000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>values</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="stop_plot2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="start_plot2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,5,1">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="connect_button">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ip_label">
             <property name="text">
              <string> IP:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="port_label">
             <property name="text">
              <string> Port:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="con_status">
             <property name="text">
              <string>Not connected</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true"/>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="start_receive">
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="stop_receive">
               <property name="text">
                <string>Stop</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Receiving?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Lost Packs:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="PlotWidget" name="graph2" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="PlotWidget" name="graph1" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Configuration</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>225</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="7" column="1">
        <widget class="QPushButton" name="refresh_config">
         <property name="text">
          <string>Refresh Configuration</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Samples per sequence:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="offset_edit"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="sps_edit">
         <property name="inputMask">
          <string>00000000</string>
         </property>
         <property name="text">
          <string>16</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Sequence repititions:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="shifts_edit">
         <property name="inputMask">
          <string>0000000</string>
         </property>
         <property name="text">
          <string>1280</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="sr_edit">
         <property name="inputMask">
          <string>000000</string>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Offset:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Shifts:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Length:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Configure Interpretation</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="length_edit">
         <item>
          <property name="text">
           <string>8160</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16080</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32640</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>64320</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>92460</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>311</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="5" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Update FPGA</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Samples per sequence:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_9"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>Shifts:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_7"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>Offset:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="spinBox_2"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>Sequence repititions:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="widget_3" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_25">
         <property name="styleSheet">
          <string notr="true">color: red
</string>
         </property>
         <property name="text">
          <string>In the making!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>20</y>
        <width>20</width>
        <height>181</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Connection</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>365</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>IP-Address:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Receiving Port:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="senport">
         <property name="inputMask">
          <string>0000</string>
         </property>
         <property name="text">
          <string>9091</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Sending Port:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="recport">
         <property name="inputMask">
          <string>0000</string>
         </property>
         <property name="text">
          <string>9090</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="refresh">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="ip1">
           <property name="inputMask">
            <string>000</string>
           </property>
           <property name="text">
            <string>192</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ip2">
           <property name="inputMask">
            <string>000</string>
           </property>
           <property name="text">
            <string>168</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ip3">
           <property name="inputMask">
            <string>000</string>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ip4">
           <property name="inputMask">
            <string>000</string>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ip1</tabstop>
  <tabstop>ip2</tabstop>
  <tabstop>ip3</tabstop>
  <tabstop>ip4</tabstop>
  <tabstop>recport</tabstop>
  <tabstop>senport</tabstop>
  <tabstop>refresh</tabstop>
  <tabstop>QComboBox_1</tabstop>
  <tabstop>QComboBox_2</tabstop>
  <tabstop>shifts_edit</tabstop>
  <tabstop>sps_edit</tabstop>
  <tabstop>offset_edit</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
